perun-base/src/main/resources/perun-roles.yml

```diff
@@ -7159,6 +7179,7 @@ perun_roles_management:
       - VOADMIN: Vo
       - VOOBSERVER: Vo
     associated_read_roles: []
+    assignable_to_attributes: false
```

Comment:
I think this should be true, we should be already able to evaluate attribute rights for SPONSOR role.